docs/docker.md

```diff
@@ -127,42 +68,36 @@ The postgres container will have created a database already, but we need
 to migrate the Django project to this database. The general way to run
 the manage.py command in docker is like this:
 
-    docker-compose -f dev.yml django python manage.py <manage command and options>
+    docker-compose exec -f dev.yml django python manage.py <manage command and options>
```

Comment:
:+1: fixed in https://github.com/DistrictDataLabs/cultivar/pull/91/commits/81ea0d3d71670aa0a863871193f1bf1d82761eaf